src/tools/build_runner.md

```diff
@@ -0,0 +1,111 @@
+---
+title: build_runner
+description: A tool for building, testing, and running Dart code.
+---
+
+The [build_runner][] package provides general-purpose commands for generating files,
+and for optionally serving or testing the generated files.
+Read this page for an overview of using build_runner, with links to
+where you can find more information.
+For details of using build_runner with a specific package,
+see the documentation for that package.
+
+<aside class="alert alert-info" markdown="1">
+  **If you're a web developer:**
+  You can use the [`webdev` tool][webdev] instead of directly using
+  build_runner to build and serve web apps.
+</aside>
+
+The build_runner commands work with _builders_—packages
+that use the [Dart build system][build]
+to generate output files from input files.
+For example, the [json_serializable][] and [built_value_generator][]
+packages define builders that generate Dart code.
+
+Although the Dart build system is a good alternative to
+[reflection][] (which has performance issues) and
+macros (which Dart's compilers don't support),
+it can do more than just read and write Dart code.
+For example, the [sass_builder][] package implements a builder that
+generates `.css` files from `.scss` and `.sass` files.
+
+
+## Setting up build_runner
+
+To use build_runner, add a [dev dependency][] on **build_runner**
+to your app's pubspec:
+
+```
+  dev_dependencies:
+    # ···
+    build_runner: ^1.0.0
+    build_test: ^0.10.3
+```
```

Comment:
Kathy: We should be able to get this via a code excerpt, no?

Reply:
> It wasn't convenient ...

Would you like for me to look into it in a followup PR?

> ... note about that being OK because this is a dev dependency

@kevmoo are you ok with such a note?

Reply:
A followup PR would be great. Thanks, @chalin!

Reply:
#1217